thoth-export-server/src/bibtex/bibtex_thoth.rs

```diff
@@ -409,6 +395,34 @@ mod tests {
                     },
                     affiliations: vec![],
                 },
+                WorkContributions {
+                    contribution_type: ContributionType::EDITOR,
+                    first_name: Some("Editor".to_string()),
+                    last_name: "2".to_string(),
+                    full_name: "Editor 3".to_string(),
+                    main_contribution: false,
+                    biography: None,
+                    contribution_ordinal: 6,
+                    contributor: WorkContributionsContributor {
+                        orcid: None,
+                        website: None,
+                    },
+                    affiliations: vec![],
+                },
+                WorkContributions {
+                    contribution_type: ContributionType::TRANSLATOR,
+                    first_name: Some("Translator".to_string()),
+                    last_name: "1".to_string(),
```

Comment:
Fix mismatched `last_name` / `full_name`s to avoid confusion (oops re the one I'd introduced above :sweat_smile: )